### PR TITLE
feat: improves jdk command output messages

### LINKED ISSUE #347: Poor `jdk` commands output
The current output of `jdk` commands (list, inst) is barebones and not always user-friendly.

For example:
* `jdk list` when no jdks are installed just prints an empty line.
* `jdk uninstall {}` prints nothing.

### PR DESCRIPTION
**Pull request by abelsromero**
Adds more friendly messages for 'jdk list' and 'jdk uninstall' adding
feedback for users.

Fixes #347

### CODE REVIEW

**Comment by maxandersen** [comment]
lookslike you made some linux assumptions in tests :)

**Comment by abelsromero** [comment]
I honestly out of ideas, I am gonna have to make a VM

**Comment by abelsromero** [comment]
fixed :partying_face:

File: src/main/java/dev/jbang/cli/Jdk.java [hunk 1]
```diff
@@ -125,4 +133,10 @@ public Integer defaultJdk(
 		}
 		return CommandLine.ExitCode.OK;
 	}
+
+	public static void main(String[] args) {
```
Comment (maxandersen) [inline src/main/java/dev/jbang/cli/Jdk.java:137]: this looks out of place - left over testing/sanity check code ?

**Review by maxandersen — APPROVED** [review]
looks fine now but please look at the "extra" main method

**Review by maxandersen — CHANGES_REQUESTED** [review]
pressed wrong button :)

**Comment by maxandersen** [comment]
i've forced pushed with rebase + remove of bad main method. if it pass test i'll merge to hopefully make release.

**Comment by abelsromero** [comment]
> i've forced pushed with rebase + remove of bad main method. if it pass test i'll merge to hopefully make release

sorry u_U I mistake

**Pull request merged**